tardis/io/tests/test_model_reader.py

```diff
@@ -123,3 +142,33 @@ def test_simple_read_cmfgen_density(cmfgen_fname):
     )
     assert len(mean_density) == 9
     assert len(velocity) == len(mean_density) + 1
+
+
+def test_csvy_finds_csv_first_line(csvy_full_fname):
+    yaml_dict, csv = load_csvy(csvy_full_fname)
+    npt.assert_almost_equal(csv["velocity"][0], 10000)
+
+
+def test_csv_colnames_equiv_datatype_fields(csvy_full_fname):
+    yaml_dict, csv = load_csvy(csvy_full_fname)
+    datatype_names = [od["name"] for od in yaml_dict["datatype"]["fields"]]
+    for key in csv.columns:
+        assert key in datatype_names
+    for name in datatype_names:
+        assert name in csv.columns
+
+
+def test_csvy_nocsv_data_is_none(csvy_nocsv_fname):
+    yaml_dict, csv = load_csvy(csvy_nocsv_fname)
+    assert csv is None
+
+
+def test_missing_required_property(csvy_missing_fname):
+    yaml_dict, csv = load_csvy(csvy_missing_fname)
+    with pytest.raises(Exception):
```

Comment:
Please be more specific about which `Exception` you are checking for.  Checking for a general Exception isn't really all that informative, and if an unexpected exception gets thrown here then the test will still pass.